openff/recharge/cli/reconstruct.py

```diff
@@ -144,4 +112,16 @@ def reconstruct(
         },
     )
 
-    esp_store.store(*esp_records)
+    with ProcessPoolExecutor(
+        max_workers=n_processors, mp_context=get_context("spawn")
+    ) as pool:
+
+        futures = [
+            pool.submit(_process_result, qc_result, grid_settings=grid_settings)
+            for qc_result in qc_results
+        ]
+        # to avoid simultaneous writing to the db, wait for each calculation
+        # to finish then write
+        for future in tqdm(as_completed(futures), total=len(futures)):
```

Comment:
Maybe add a description to the progress bar so we know what stage this is like `Calculating ESP`